apps/angular/src/core/pipes/no-empty.pipe.ts

```diff
@@ -0,0 +1,19 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+const NULL_VALUE = '-';
```

Comment:
This name can be improve, cause one read it, they will assume `NULL_VALUE = null` not `-`. Try this
```suggestion
const NULL_VALUE_FALLBACK_LABEL = '-';
```